src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesLineObsolete.TableExt.al

```diff
@@ -98,20 +98,20 @@ tableextension 309 NoSeriesLineObsolete extends "No. Series Line"
         exit(NoSeries.PeekNextNo(Rec, WorkDate()));
     end;
 
-    [Obsolete('This functionality has been removed and getting the number from a string is no longer part of No. Series.', '24.0')]
+    [Obsolete('This functionality has been made internal.', '24.0')]
```

Comment:
As a developer getting this warning, I could not determine what to do with it. As I derive from the code, this procedure `ExtractNoFromCode` itself has not been made internal - it can be still called from outside. Same for the `GetFormattedNo` procedure.

You probably meant that this procedure will be removed later without public replacement?

Reply:
yes, the method will no longer be available. I'll update the message.